src/tool/python/file_writers.h

```diff
@@ -22,28 +23,26 @@ namespace xlang
             w.needed_namespaces.insert(needed_ns);
         }
 
-        xlang::filter f{ settings.include, settings.exclude };
         auto filename = w.write_temp("py.%.h", ns);
 
-        f.bind_each<write_delegate>(members.delegates)(w);
-        f.bind_each<write_pinterface_decl>(members.interfaces)(w);
-        f.bind_each<write_pinterface_impl>(members.interfaces)(w);
+        settings.filter.bind_each<write_delegate_callable_wrapper>(members.delegates)(w);
+        settings.filter.bind_each<write_pinterface>(members.interfaces)(w);
 
         w.write("\nnamespace py\n{\n");
         {
             writer::indent_guard g{ w };
-            f.bind_each<write_winrt_type_specialization>(members.classes)(w);
-            f.bind_each<write_winrt_type_specialization>(members.interfaces)(w);
-            f.bind_each<write_winrt_type_specialization>(members.structs)(w);
-            f.bind_each<write_struct_converter_decl>(members.structs)(w);
-            f.bind_each<write_pinterface_type_mapper>(members.interfaces)(w);
-            f.bind_each<write_delegate_type_mapper>(members.delegates)(w);
+            settings.filter.bind_each<write_get_python_type_specialization>(members.classes)(w);
```

Comment:
This pattern is a bit weird. No change needed, but I'll probably try to harmonize the various bind functions soon.